### ISSUE TITLE
Package name collision error when updating bosh from 264.2.0 to 264.7.0

### ISSUE THREAD

**Issue by penrod** [cloudfoundry/bosh#1889]
It looks like the fix for the package name collision broke somewhere between 264.2.0 and 264.7.0. I am attempting to update from bosh 264.2.0 to bosh 264.7.0 and its failing with: 

'Error: Package name collision detected in instance group 'bosh': job 'bosh/nats' depends on package 'bosh/ruby-2.4-r3', job 'bosh-vsphere-cpi/vsphere_cpi' depends on 'bosh-vsphere-cpi/ruby-2.4-r3'. BOSH cannot currently collocate two packages with identical names from separate releases.'

Checking the release notes, 264.2.0 has the fix for this error, but when updating from 264.2.0 to 264.7.0, the error returns. 

Deploying:
- name: bosh
  version: "264.7.0"
- name: bosh-vsphere-cpi
  version: "45.1.0"
vsphere stemcell 3468.20

https://github.com/cloudfoundry/bosh/releases/tag/v264.2.0

**Comment by dpb587-pivotal** [comment]
Sounds like you may be deploying BOSH via another BOSH. Has the "outer" BOSH which is deploying the other BOSH been upgraded past 264.2.0 which fixes the issue?

**Comment by penrod** [comment]
I suspected that may be the case but the part that didn't make sense to me was that I was able to update bosh from 263.3 to 264.2 with no problem. If I attempted to update 264.2 to 264.7, I would then get the name collision error. 

If you are curious, we can dig deeper on this but at this point I am going to close this as I ran into issues with the nats certs needed for the 264.1 update and will not be updating the bosh versions until I can figure that one out.  I'll re-open it if the issues persists once I update the deploying bosh to 254.2. 

Thanks
